docs/jailer.md

```diff
@@ -279,6 +279,11 @@ Note: default value for `<api-sock>` is `/run/firecracker.socket`.
 
 ### Known limitations
 
+- Time it takes to create a jail depends on the number of mount points in the
+  system and the number of jailers starting at the same time. The approximate
+  slowdown of jail creation time is 2x when 10 jails are created in parallel and
```

Comment:
I'm finding this a bit hard to read. 2x when 10 jails and 10x when 10 jails? Is 500 mount points applicable to the latter only? Then how many are for the former?

Reply:
updated